vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SampleRegistrationSpreadsheet.java

```diff
@@ -378,9 +378,8 @@ private void generateGenomicsSheet() {
    * @return cellValueOptionsForColumnMap map grouping the selectable cell values within the cells
    * of a column with the {@link SamplesheetHeaderName} of the colum
    */
-  private LinkedHashMap<SamplesheetHeaderName, List<String>> cellValueOptionsForCellWithinColumn(
+  private void generateCellValueOptionsMap(
```

Comment:
Is this correctly reset when an experiment is changed?

Reply:
Yes! 🙌